server/src/main/java/org/opensearch/repositories/ShardGenerations.java

```diff
@@ -60,6 +60,8 @@ public final class ShardGenerations {
      */
     public static final String NEW_SHARD_GEN = "_new";
 
+    public static final String REMOTE_STORE_INTEROP_SHARD_GEN_SUFFIX = "_rs";
```

Comment:
What is this used for ?

Reply:
removed it, not needed anymore with current approach.